build/Build.proj

```diff
@@ -81,7 +81,7 @@
     <!-- Only build the following clients if the SDK is installed -->
     <Projects Include="$(ProjectRoot)\src\Microsoft.AspNet.SignalR.Client.Portable\Microsoft.AspNet.SignalR.Client.Portable.csproj">
       <Build Condition="!$(WP8SDKInstalled)">false</Build>
-      <Platform>portable-net45+sl5+netcore45+wp8+wp81</Platform>
+      <Platform>portable-net45+sl5+netcore45+wp8+wp81+win81+wpa81</Platform>
```

Comment:
Actually, something smells a little fishy here. `win81` is supposed to map to `netcore451` so it should already be compatible with the old provide (via `netcore45`).

When I create a portable project targeting all these frameworks and pack it I get: `portable-net45+sl50+win+wpa81+wp80` which maps to `.NETPortable,Version=v4.0,Profile=Profile344`.

Can you try that name and see if it fixes the bug? If it does, that should resolve the issue and ensure we are continuing to use "real" portable profiles.